ext/dom/tests/bug47849.phpt

```diff
@@ -19,4 +19,4 @@ echo $aDOM->saveXML();
 ?>
 --EXPECT--
 <?xml version="1.0"?>
-<r:root xmlns:r="urn::root"><data xmlns="urn::data"/></r:root>
+<r:root xmlns:r="urn::root"><data xmlns="urn::data">aaa</data></r:root>
```

Comment:
@beberlei as you've touched the DOM extension recently, can you just confirmed that the displayed behaviour is correct and that the current test is bogus or is the test as it is correct and I need to investigate why this change happens.

Reply:
@Girgias i imagine its the recursive = 2 assignment added by this patch: https://github.com/php/php-src/commit/408a1746a10cb35bcee403458f87d762ab7aeb0f

You need to convert the "recursive" bool into a new `int` because that is what xmlDocCopyNode accepts